debug/terrain_control.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+<head>
+   <title>MapLibre GL JS debug page</title>
+   <meta charset='utf-8'>
+   <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
+   <link rel='stylesheet' href='../dist/maplibre-gl.css' />
+   <style>
+       body { margin: 0; padding: 0; }
+       html, body, #map { height: 100%; }
+   </style>
+</head>
+
+<body>
+<div id='map'></div>
+
+<script src='../dist/maplibre-gl-dev.js'></script>
+<link href='../dist/maplibre-gl.css' rel='stylesheet' />
+<script>
+
+var map = window.map = new maplibregl.Map({
+    container: 'map',
+    zoom: 12.5,
+    center: [11.40416, 47.26475],
+    hash: true,
+    style: "https://static.maptoolkit.net/styles/toursprung/terrain.json",
+    maxZoom: 16
+});
+
+map.on("load", () => {
+  map.addSource("terrain", {
+     "type": "raster-dem",
+     "tiles": ["https://vtc-cdn.maptoolkit.net/terrainrgb/{z}/{x}/{y}.webp"],
+     "encoding": "mtk",
+     "maxzoom": 14
+  });
+  map.addControl(
+    new maplibregl.TerrainControl({
+      id: "terrain"
```

Comment:
This line is missing a comma, it should be 
`      id: "terrain",`

Reply:
@prozessor13 just wanted to mentions this. I noticed there is a missing comma in my example that was merged in.